scripts/rust-analyze.sh

```diff
@@ -14,10 +14,12 @@ CONFIG_FILE=$(realpath $1)
 TREE_NAME=$2
 
 if [ -d "$OBJDIR" ]; then
-  ANALYSIS_DIRS="$(find $OBJDIR -type d -name save-analysis)"
+  ANALYSIS_DIRS="$(find $OBJDIR -type d -name save-analysis) $(find `rustc --print sysroot` -type d -name analysis)"
```

Comment:
We should probably do a thing here where we first check for the save-analysis files in $OBJDIR, and if there aren't any, we bail out. Otherwise we'll run the rust-indexer even on repos with no rust code (e.g. for nss) and just "analyze" the stdlib analysis files.